### ISSUE TITLE
No visual cue on project import

### ISSUE THREAD

**Issue by dnabeast** [bdkjones/CodeKit#43]
**Quick, short summary:**
When importing a new project there is no visual cue to show that the import is occuring.

**Expected results:**
Drag a medium sized project onto the codekit window (Add as a project) and a spinner or loading bar animates while the files are imported.

**Actual results:**
Screen returns to pre import state and appears static. Package import displays down bottom and what appears to be a loading bar has a static image.
![pulse](https://cloud.githubusercontent.com/assets/3229763/15459025/30612200-20e4-11e6-8679-580e82588d67.png)

A few seconds later the project loads and refreshes the page.

**Exact steps to reproduce:**
Drag new project onto (Add as a Project)

I only mention this because my first instinct when it happened was that it had done so really quickly and then I checked the list of imported projects and it wasn't there and I thought it had failed. What had actually occurred was that it was still loading.

**Comment by dnabeast** [comment]
I am not seeing this behaviour. Even with codekit frontmost the progress bar remains static and grey on import.

Also, when dragging projects across I would expect in 99% of instances the finder would be the frontmost app.

**Comment by bdkjones** [comment]
Screen recording please. Make sure it has the entire screen, including the menubar at the top.

**Comment by dnabeast** [comment]
![codekitpulse](https://cloud.githubusercontent.com/assets/3229763/20287778/51d94b0e-ab1a-11e6-8d79-3028510675ef.gif)

Okay. I may have spotted the issue. When I reviewed the gif I was able to make out the very subtle flowing of lighter grey into lighter grey in the progress bar. I went back to the original program and could just discern some movement. I've attached the gif none-the-less to make sure this it correct behaviour as I didn't detect it when looking for it so could be a usability bug if it's not a programming one ;)
